packages-ts/starknet-gauntlet-oz/src/commands/account/deploy.ts

```diff
@@ -47,7 +52,23 @@ const beforeExecute: BeforeExecute<UserInput, ContractInput> = (context, input,
 }
 
 const afterExecute: AfterExecute<UserInput, ContractInput> = (context, input, deps) => async (result) => {
-  deps.logger.success(`Account contract located at ${result.responses[0].tx.address}`)
+  const contract = result.responses[0].tx.address
+  contract
+    ? deps.logger.success(`Account contract located at ${contract}`)
+    : deps.logger.error('Account contract not successfully created')
+
+  if (input.user.salt != undefined) {
+    const calcAddr = calculateAddress(input.user.salt, input.user.publicKey)
+
+    // log error if address mismatch
+    if (!equalAddress(contract, calcAddr)) {
+      deps.logger.error(`Deployed account (${contract}) does not match calculated account (${calcAddr})`)
+      deps.logger.warn(`Account addresses must match otherwise this could cause mismatched keys with chainlink node`)
```

Comment:
Maybe rephrase this so that it's clear what keys mismatch and what's the impact?